ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,34 @@
+<!--- Provide a general summary of the issue in the Title above -->
```

Comment:
There was 2 choices in the template generator I used to quick-start this: one was "template focused on issue", the other one was "template focused on improvement". I chose to focus on issues, because I feel it's usually the hardest to decrypt from reporters. That being said, we should definitely say something about improvement yes.

Or maybe have each section be phrased to take both possibilities into account? I'll try to see if it can be done easily (and without too much verbosity).